migrations/1-create-order.js

```diff
@@ -0,0 +1,39 @@
+'use strict'
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('order', {
```

Comment:
if you're standardizing on lowercase for table names, that's fine, just be consistent. Some people prefer to caps the first letter, called PascalCase, but if you'd rather do it this way it's ok. Just want to point out the implicit decision being made here.